contrib/inventory/ec2.py

```diff
@@ -491,9 +491,14 @@ def get_rds_instances_by_region(self, region):
         try:
             conn = self.connect_to_aws(rds, region)
             if conn:
-                instances = conn.get_all_dbinstances()
-                for instance in instances:
-                    self.add_rds_instance(instance, region)
+                marker = None
+                while True:
+                    instances = conn.get_all_dbinstances(marker=marker)
+                    marker = instances.next_marker
```

Comment:
@Sodki , from docs it is supposed to be 'next_marker' as marker returns what i passed in, but this might be the issue https://github.com/boto/boto/issues/3109

as for marker, i even looked at the code:

`:param marker: The marker provided by a previous request.`